dev/com.ibm.ws.jaxws.tools/bnd.bnd

```diff
@@ -24,9 +24,9 @@ Export-Package: com.sun.tools.ws.ant;version="2.2.10", \
  org.jvnet.staxex;version="1.7.7", \
  org.jvnet.staxex.util;version="1.7.7"
 
-Import-Package: \
- com.sun.xml.bind*;version="2.2.10"
-
+DynamicImport-Package: \
```

Comment:
Why doing dynamic import?  The transforms I did take care of this right?

Reply:
Yeah, that was a hack I didn't intend to keep when I merged your branch.